source/common/upstream/resource_manager_impl.h

```diff
@@ -25,7 +25,7 @@ struct ManagedResourceImpl : public BasicResourceLimitImpl {
   }
 
   // BasicResourceLimitImpl
-  bool canCreate() override { return current_ < max(); }
+  bool canCreate() override { return BasicResourceLimitImpl::canCreate(); }
```

Comment:
Hi, if this could work, why not use `BasicResourceLimitImpl::canCreate` directly?

Reply:
HI, frankly i'm not sure why not use the parent api directly, maybe because of some design decisions? I just found the method (kind of) repeating its parent and made a simple suggestion. :)

Reply:
I think we should just remove this since it's the same implementation as the parent.